tests/test_autonomy/test_cli/test_helpers/test_docstrings.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2022 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""The module contains test_helpers for module tests."""
+
+import shutil
+import tempfile
+from pathlib import Path
+from types import ModuleType
+from typing import cast
+from unittest import mock
+
+import pytest
+from _pytest.capture import CaptureFixture  # type: ignore
+
+from autonomy.cli.helpers.docstring import analyse_docstrings, import_rounds_module
+
+import packages
+from packages.valory.skills import test_abci
+from packages.valory.skills.test_abci import rounds as test_abci_rounds
+
+
+@pytest.mark.parametrize("module", [test_abci, test_abci_rounds])
+def test_import_rounds_module(module: ModuleType) -> None:
+    """Test import_rounds_module"""
+
+    module_path = Path(module.__file__)
+    module = import_rounds_module(module_path)
+    assert module is test_abci_rounds
+
+
+def test_import_rounds_module_failure() -> None:
+    """Test import_rounds_module"""
+
+    with pytest.raises(ModuleNotFoundError, match="No module named 'packages.rounds'"):
+        module_path = Path(packages.__file__)
+        import_rounds_module(module_path)
+
+
+def test_import_rounds_module_with_non_default_package_dir() -> None:
+    """Test import_rounds_module from different package directory"""
+
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        tmp_path = Path(tmp_dir)
+        module_path = Path(packages.__file__)
+        shutil.copytree(module_path.parent, tmp_path / packages.__name__)
+        new_package_dir = tmp_path / module_path.parts[-1]
+        module = import_rounds_module(module_path, packages_dir=new_package_dir)
+        assert module.__file__.startswith(tmp_dir)
+
+
+@pytest.mark.parametrize("module", [test_abci, test_abci_rounds])
+def test_analyse_docstrings_without_update(module: ModuleType) -> None:
+    """Test analyse_docstrings"""
+
+    module_path = Path(module.__file__)
+    updated_needed = analyse_docstrings(module_path)
+    assert not updated_needed
```

Comment:
this mimics the input in the first test above, `test_import_rounds_module`, but will fail and return `True` for the `test_abci` module because it reads the from the passing in `module_path`, which does not match the retrieved module from the `import_rounds_module`. 
https://github.com/valory-xyz/open-autonomy/blob/c3c82f868bcff341e31d4ea222a1448fdcc6eaf0/autonomy/cli/helpers/docstring.py#L48-L69